plugins/carbon_black_cloud/icon_carbon_black_cloud/tasks/monitor_alerts/task.py

```diff
@@ -154,18 +173,21 @@ def trigger_observation_search_job(
         self, start_time: str, end_time: str, page_size: int, debug: bool, state: Dict[str, str]
     ) -> Tuple[str, Dict]:
         endpoint = f"api/investigate/v2/orgs/{self.connection.org_key}/observations/search_jobs"
+        index = state.get(OBSERVATION_JOB_OFFSET, 0)
         search_params = {
             "rows": page_size,
-            "start": 0,
+            "start": index,
             "fields": ["*"],
             "criteria": {"observation_type": OBSERVATION_TYPES},
             "sort": [{"field": OBSERVATION_TIME_FIELD, "order": "asc"}],
             "time_range": {"start": start_time, "end": end_time},
         }
         url = f"{self.connection.base_url}/{endpoint}"
-        self.logger.info(f"Triggering observation search using parameters {search_params['time_range']}")
+        self.logger.info("Triggering observation search")
```

Comment:
Interesting case with that `structlog` here. Imo, for know to pass the unittests check you can f-string all the parameters you want instead of putting them as kwargs. More or less like this
```suggestion
        self.logger.info(f"Triggering observation search {start_time=} | {end_time=} | {index=}")
```

Reply:
Fixed the CI so unit tests pass now without this